src/install.ts

```diff
@@ -23,13 +23,13 @@ export async function install(platform: string, architecture: string, release: s
     // Install runtime system dependencies for MATLAB on Linux
     if (platform === "linux") {
         await core.group("Preparing system for MATLAB", () =>
-            script.downloadAndRunScript(platform, properties.matlabDepsUrl, [version.release])
+            script.downloadAndRunScript(platform, properties.matlabDepsUrl, [version.release.substring(0,6)])
```

Comment:
How about making version.release the actual release name without the update number, since we already parsed and stored update in a separate field. Then we can avoid doing substring outside of the version parsing function.

Reply:
updated